src/components/space/functionality/ViewSelect.js

```diff
@@ -37,19 +38,19 @@ const ViewSelect = () => {
     setView(userSelection.value);
   };
 
+  let viewMessage = '';
+  if (view === BUILDER_VIEW_STRING) {
+    viewMessage =
+      "The 'builder' view displays analytics from the default Graasp item creation interface.";
+  } else if (view === PLAYER_VIEW_STRING) {
+    viewMessage =
+      "The 'player' view displays analytics from the standalone Graasp interface typically used by students to access an item.";
+  } else if (view === EXPLORER_VIEW_STRING) {
+    viewMessage =
+      "The 'explore' view displays analytics from the standalone Graasp interface typically used by visualize resources.";
```

Comment:
`"The 'explorer' view displays analytics from the Graasp interface sharing open educational resources."`

Reply:
Solved